src/weekly/input.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import moment from 'moment'
+
+export type input = {
+  deliveryDate: ?string,
+  deliveryDayOfWeek: ?string,
+  minDaysInAdvance: ?number
+}
+
+let weekDays:Map<string, number> = new Map([
+  ['SUNDAY', 0],
+  ['MONDAY', 1],
+  ['TUESDAY', 2],
+  ['WEDNESDAY', 3],
+  ['THURSDAY', 4],
+  ['FRIDAY', 5],
+  ['SATURDAY', 6],
+  ['SUN', 0],
+  ['MON', 1],
+  ['TUE', 2],
+  ['WED', 3],
+  ['THU', 4],
+  ['FRI', 5],
+  ['SAT', 6]
+])
+
+export function getDeliveryDate (input: input) {
```

Comment:
I'm not keen on having a type and a parameter with the same name (if thats what this is?)